Source/RomanNumeral.c

```diff
@@ -13,4 +13,24 @@ static char * Append(char *output, const char *toAppend)
 
 void RomanNumeral_Convert(unsigned input, char *output)
 {
+   char *CharactersArray []= {"M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"};
+   unsigned DecimalArray []= {1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1};
```

Comment:
Can you think of a way to "tie" these values into a single container? ~hint: struct~